app-frontend/src/app/core/services/layer.service.js

```diff
@@ -207,12 +207,19 @@ export default (app) => {
                 redGamma: object.redGamma,
                 greenGamma: object.greenGamma,
                 blueGamma: object.blueGamma,
+                redMax: object.redMax,
```

Comment:
It's definitely not, I'm going to nuke it in the storage refactor (along with lots of other stuff :tada: ).